share/spice/just_delete_me/content.handlebars

```diff
@@ -3,13 +3,8 @@
     {{notes}}
   </p>
 {{/if}}
-{{#if delete_url}}
```

Comment:
Again, I must object to "Account Deletion Page" as the link does not always link to the page to delete the account.